README.md

```diff
@@ -92,10 +109,10 @@ try command ```kind load docker-image <image-name>:<tag> ``` for each of the ima
 Secondly, install helm chart.
 
 ```
-helm install installer/chart --namespace <namespace> --name <specified-name>
+helm install installer/chart/volcano --namespace <namespace> --name <specified-name>
```

Comment:
the folder `/installer/chart/volcano` has been removed in the master branch, just asking to ensure you are adding this back for purpose.

Reply:
Now we charts under, installer/chart, since chart name and folder it is kept should be same to generate chart package, have create volcano folder inside installer/chart and have moved all those required files inside it.  So I just have moved charts from installer/chart to installer/chart/volcano.  So have made those changes in readme and script file too.

Reply:
sure